hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -1146,7 +1202,7 @@ public HoodieTimeline getTimeline() {
   @Override
   public void sync() {
     HoodieTimeline oldTimeline = getTimeline();
-    HoodieTimeline newTimeline = metaClient.reloadActiveTimeline().filterCompletedAndCompactionInstants();
+    HoodieTimeline newTimeline = metaClient.reloadActiveTimeline().filterCompletedInstantsOrRewriteTimeline();
```

Comment:
not sure why we have renamed this. also, curious as to why we have added clustering to the filtering list. prior to this patch, this method was filtering for any completed instant or if its compaction. Now, w/ this patch, I would expect, any completed instant or if its compaction or log compaction.